SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj

```diff
@@ -14,16 +14,17 @@
   <Import Project="..\Microsoft.AppCenter.Shared\Microsoft.AppCenter.Shared.projitems" Label="Shared" />
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="sqlite-net-pcl" Version="1.3.1" />
+    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.14" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
     <PackageReference Include="System.Management" Version="4.6.0" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.6.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net45'">
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Management" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Deployment" />
+    <Reference Include="System.Management" />
+    <Reference Include="System.Net.Http" />
```

Comment:
.net core is not impacted by missing http?

Reply:
Why the need for that in the first place?

Reply:
It was already discussed: .net core doesn't seem affected, need - it fails without it (It was referenced implicitly before somehow).
We'll recheck it again.

Reply:
Also, it's not new topic. Some users were affected by this: https://github.com/microsoft/appcenter-sdk-dotnet/issues/1134#issuecomment-549751706

I don't think that it affects core build, but we should add a requirement of it for .net framework in nuspecs

Reply:
Created a separate PR with the fix (to develop) since it is NOT related to this feature.
See #1251